.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 max-line-length = 100
+extend-ignore = E203, E501, E704
```

Comment:
Any reason why you can't use Black to auto-format anything violating these rules?

Reply:
Its what black formatter recommended config for flake8. I couldn't really add in the other configs that black formatter recommended for flake8 because it would fail all the python files otherwise